Unstable Jenkins e2e test phase

The end to end Protractor test phase breaks often on the CI server.
The problem is not reproducible on a local environment.

```
[12:11:48] E/launcher - waiting for element '.last-block-loaded'
Wait timed out after 4005ms
[12:11:48] E/launcher - TimeoutError: waiting for element '.last-block-loaded'
Wait timed out after 4005ms
    at WebDriverError (/home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/selenium-webdriver/lib/error.js:27:5)
    at TimeoutError (/home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/selenium-webdriver/lib/error.js:238:5)
    at /home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/selenium-webdriver/lib/promise.js:2107:17
    at ManagedPromise.invokeCallback_ (/home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/selenium-webdriver/lib/promise.js:1366:14)
    at TaskQueue.execute_ (/home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/selenium-webdriver/lib/promise.js:2970:14)
    at TaskQueue.executeNext_ (/home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/selenium-webdriver/lib/promise.js:2953:27)
    at asyncRun (/home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/selenium-webdriver/lib/promise.js:2813:27)
    at /home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/selenium-webdriver/lib/promise.js:676:7
    at process._tickCallback (internal/process/next_tick.js:109:7)
From: Task: waiting for element '.last-block-loaded'
    at scheduleWait (/home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/selenium-webdriver/lib/promise.js:2094:20)
    at ControlFlow.wait (/home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/selenium-webdriver/lib/promise.js:2408:12)
    at Driver.wait (/home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/selenium-webdriver/lib/webdriver.js:943:29)
    at run (/home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/protractor/lib/browser.ts:66:27)
    at ProtractorBrowser.to.(anonymous function) [as wait] (/home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/node_modules/protractor/lib/browser.ts:74:12)
    at World.Then (/home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/features/step_definitions/generic.step.js:125:11)
[12:11:48] E/launcher - Process exited with error code 199

npm ERR! Linux 4.4.0-109-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "e2e-test" "--" "--params.baseURL" "http://localhost:6042"
npm ERR! node v6.12.3
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! lisk-explorer@1.4.1 e2e-test: `protractor protractor.conf.js "--params.baseURL" "http://localhost:6042"`
npm ERR! Exit status 199
npm ERR! 
npm ERR! Failed at the lisk-explorer@1.4.1 e2e-test script 'protractor protractor.conf.js "--params.baseURL" "http://localhost:6042"'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the lisk-explorer package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     protractor protractor.conf.js "--params.baseURL" "http://localhost:6042"
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs lisk-explorer
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls lisk-explorer
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/lisk/workspace/lisk-explorer_PR-388-PFI622ZPDPDNWKWYG4W7Z2LOYSDVM5DLQKSWYYMDZZXYPS3QFIEA/npm-debug.log
```

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -99,6 +99,21 @@ pipeline {
 				'''
 			}
 		}
+		stage ('Run e2e tests') {
+			steps {
+				sh '''
+				N=${EXECUTOR_NUMBER:-0}
+
+				# End to End test configuration
+				export DISPLAY=:9$N
+				Xvfb :9$N -ac -screen 0 1280x1024x24 &
+				./node_modules/protractor/bin/webdriver-manager start --seleniumPort 443$N &
+
+				# Run E2E Tests
+				npm run e2e -- --params.baseURL http://localhost:604$N
+				'''
+			}
+		}
 	}
 	post {
 		success {

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -91,7 +91,7 @@ pipeline {
 				'''
 			}
 		}
-		stage ('Run tests') {
+		stage ('Run API tests') {
 			steps {
 				sh '''
 				sed -i -r -e "s/6040/$EXPLORER_PORT/" test/node.js
@@ -99,19 +99,13 @@ pipeline {
 				'''
 			}
 		}
-		stage ('Run e2e tests') {
+		stage ('Run E2E tests') {
 			steps {
-				sh '''
-				N=${EXECUTOR_NUMBER:-0}
-
-				# End to End test configuration
-				export DISPLAY=:9$N
-				Xvfb :9$N -ac -screen 0 1280x1024x24 &
-				./node_modules/protractor/bin/webdriver-manager start --seleniumPort 443$N &
-
-				# Run E2E Tests
-				npm run e2e -- --params.baseURL http://localhost:604$N
-				'''
+				wrap([$class: 'Xvfb']) {
+					sh '''
+					npm run e2e -- --params.baseURL http://localhost:$EXPLORER_PORT
+					'''
+				}
 			}
 		}
 	}

--- a/features/step_definitions/generic.step.js
+++ b/features/step_definitions/generic.step.js
@@ -41,7 +41,7 @@ defineSupportCode(({ Given, When, Then, setDefaultTimeout }) => {
 
 	Given('I\'m on page "{pageAddress}"', (pageAddress, callback) => {
 		browser.ignoreSynchronization = true;
-		browser.driver.manage().window().setSize(1200, 1000);
+		browser.driver.manage().window().setSize(1280, 1024);
 		browser.driver.get('about:blank');
 		browser.get(baseURL + pageAddress).then(callback);
 	});

--- a/package.json
+++ b/package.json
@@ -13,8 +13,8 @@
   "scripts": {
     "test": "./node_modules/.bin/grunt test --verbose",
     "test-live": "npm test -- --auto-watch --no-single-run",
-    "e2e-test": "protractor protractor.conf.js",
-    "e2e-test-env-update": "./node_modules/protractor/bin/webdriver-manager update",
+    "pree2e": "./node_modules/protractor/bin/webdriver-manager update --standalone false --gecko false",
+    "e2e": "protractor protractor.conf.js",
     "eslint": "eslint ./benchmarks ./utils ./libs ./api ./src ./test ./features ./tasks ./*.js ./*.json",
     "eslint-fix": "eslint --fix ./benchmarks ./utils ./libs ./api ./src ./test ./features ./tasks ./*.js",
     "start": "npm run copy-files && webpack -w",

